### PR TITLE
Fixed year published error, and added persistence on Reviewed On Date

### PR DESCRIPTION
**Pull request by HiroyukiAbhi**
There was an error when uploading bulk content using bulk add content.
Error: String cannot be passed as int
        Cause: Year published had been written as a date not as a year
Fixed: Year published can now be received as full date
Added:
-Paper elevation to library version file view
-Reviewed On date persistence - Issue #67 
       https://github.com/SolarSPELL-Main/solarspell-dlms/issues/67

### CODE REVIEW

File: content_management/utils.py [hunk 1]
```diff
@@ -54,13 +54,25 @@ def upload_sheet_contents(self, sheet_contents):
                                 content.display_title = each_content.get("Title")
                             content.description = each_content.get("Description")
                             content.copyright_notes = each_content.get("Copyright Notes")
-                            content.reviewed_on = datetime.datetime.now()
+                           
+                            if each_content.get("Reviewed On"):
+                                try:
+                                    content.reviewed_on = datetime.datetime.strptime(each_content.get("Reviewed On"), "%m/%d/%Y, %H:%M:%S")
+                                except ValueError:
+                                    content.reviewed_on = datetime.datetime.now()
```
Comment (JacobShaeffer) [inline content_management/utils.py]: why is this line datetime.now() while the following one is datetime.now().date()?

File: content_management/utils.py [hunk 2]
```diff
@@ -132,16 +144,26 @@ def upload_sheet_contents(self, sheet_contents):
...
                             content.copyright_notes = each_content.get("Copyright Notes")
-                            content.reviewed_on = datetime.datetime.now()
-                            content.rights_statement = each_content.get("Rights Statement")
-                            if each_content.get("Year Published"):
+                            if each_content.get("Reviewed On"):
                                 try:
-                                    content.published_date = datetime.date(each_content.get("Year Published"), 1, 1)
+                                    content.reviewed_on = datetime.datetime.strptime(each_content.get("Reviewed On"), "%m/%d/%Y, %H:%M:%S").date()
+                                except ValueError:
+                                    content.reviewed_on = datetime.datetime.now()
```
Comment (JacobShaeffer) [inline content_management/utils.py]: Same question here. datetime.now() vs datetime.now().date()

**Review by JacobShaeffer — COMMENTED** [review]
Please review the 2 comments I left.

File: content_management/utils.py (hunk 2 above)
Comment (HiroyukiAbhi) [reply content_management/utils.py]: You are right, i must've missed them, they should all have .date() to convert the datetime object to a _Date object

Comment (HiroyukiAbhi) [reply content_management/utils.py]: I'll fix this right away

**Comment by HiroyukiAbhi** [comment]
Fixed potential error:
   All datetime objects have been converted to _Date objects

**Comment by JacobShaeffer** [comment]
Looks good.

**Pull request merged**